libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -317,6 +326,7 @@ instance (ToSchema cfg, IsFeatureConfig cfg) => ToSchema (WithStatusNoLock cfg)
       WithStatusNoLock
         <$> wssStatus .= field "status" schema
         <*> wssConfig .= objectSchema @cfg
+        <*> wssTTL .= field "ttl" schema
```

Comment:
```suggestion
        <*> wssTTL .= fromMaybe FeatureTTLUnlimited <$> field "ttl" schema
```

If you this this, you can drop it from the `values.yaml` above, which I really want to see (so many places where that would have to be kept in mind, and so many people involved, and it really doesn't belong there in the first place.)